app/bundles/CoreBundle/Assets/js/libraries/filemanager/index.html.twig

```diff
@@ -57,6 +57,11 @@
         $window.resize();
         {% endif %}
     });
+    window.addEventListener('beforeunload', function (e) {
```

Comment:
@adiux  This is reason why we need Mautic.builder variable. Do you have another solution for close GJS filemanager from window popup?

Based on docs https://grapesjs.com/docs/modules/Assets.html#customization

Reply:
thanks @kuzmany for looking into this. I'm not sure what you need to close and when. I guess when I double click on an image in the GJS editor the function `openServerBrowser` opens the filemanager:
<img width="968" alt="image" src="https://user-images.githubusercontent.com/13075514/203320715-816620c6-0d90-439a-a6f3-b186b803599e.png">

and then you close it by selecting an image or closing the popup. I assume there is nothing in the UI from GJS?

Reply:
ah.. I see. I had a similar problem with the commands. They need to be stopped too or they won't run again. Maybe that is related: https://grapesjs.com/docs/modules/Commands.html#stateful-commands

could it be enought if we simply specify the close event:
<img width="768" alt="image" src="https://user-images.githubusercontent.com/13075514/203525484-364d2b8d-a3f2-4aaf-9d69-350846bc5339.png">

```js
      custom: {
        open(props) {
          //...
        },
        close(props) {
          am.close()
        },
      },
```

Reply:
No it's not enough.  This nod cover situation on close window. We need say on close window to assetManager that was close. Then you can use it